test/TestFITS.cc

```diff
@@ -125,14 +125,14 @@ TEST(Filtering, FailOnWrongFilterType) {
     EXPECT_FALSE(table.View().StringFilter(table["dummy"], "N 224"));
     EXPECT_FALSE(table.View().StringFilter(table["RA"], "N 224"));
 
-    EXPECT_FALSE(table.View().NumericFilter(table["dummy"], RANGE_INCLUSIVE, 0, 100));
-    EXPECT_FALSE(table.View().NumericFilter(table["Name"], RANGE_INCLUSIVE, 0, 100));
+    EXPECT_FALSE(table.View().NumericFilter(table["dummy"], CARTA::RangeOpen, 0, 100));
```

Comment:
Has this deliberately been changed from closed to open?

Reply:
good catch, it should also be `RangeClosed`. Fixed